kolibri/plugins/management/assets/src/vue/user-page/index.vue

```diff
@@ -198,10 +202,29 @@
     },
     $trNameSpace: 'userPage',
     $trs: {
-      noUsersExist: 'No Users Exist.',
-      allUsersFilteredOut: 'No users match the filter.',
+      // input & accessibility labels
```

Comment:
@indirectlylit I feel like being that verbose would gunk up the markup to the point that reading though it would get pretty frustrating, but if it helps out translators, I can see an argument for it.

One thing I've been thinking about: Do all of these these strings get consolidated from a translator's perspective? So if `Admin` is used in `Learn` and `Manage`, do they have to translate it twice?